release/tools/build.xml

```diff
@@ -171,8 +189,28 @@
         <property name="build.num" value="build_${time.stamp.bundle.string}"/>
         <property name="bundle.dir" value="${build.dir}/${build.num}/bundle"/>
         
-        <property name="deliverable.bundle.dir" value="${bundle.dir}/${deliverable.name.token}${deliverable.type}"/>
-        <property name="zip.name" value="${deliverable.name.token}${deliverable.type}"/>
+        <!--property name="deliverable.bundle.dir" value="${bundle.dir}/${deliverable.name.token}${deliverable.type}"/-->
+        <condition property="deliverable.bundle.dir" value="${bundle.dir}/${deliverable.name.token}${deliverable.type}">
+            <contains string="${deliverable.name}" substring="jakartaee" casesensitive="false"/>
+        </condition>
+        <condition property="deliverable.bundle.dir" value="${bundle.dir}/${deliverable.tck.dir}-${deliverable.type}">
+            <not>
+                <contains string="${deliverable.name}" substring="jakartaee" casesensitive="false"/>
+            </not>
+        </condition>
+
+	<!--property name="deliverable.bundle.dir" value="${bundle.dir}/${deliverable.tck.dir}-${deliverable.type}"/-->
+
+        <!--property name="zip.name" value="${deliverable.tck.dir.token}-${deliverable.type}"/-->
+	<condition property="zip.name" value="${deliverable.name.token}${deliverable.type}">
+            <contains string="${deliverable.name}" substring="jakartaee" casesensitive="false"/>
+        </condition>
+        <condition property="zip.name" value="${deliverable.tck.dir.token}-${deliverable.type}">
+            <not>
+                <contains string="${deliverable.name}" substring="jakartaee" casesensitive="false"/>
+            </not>
+        </condition>
+
```

Comment:
I can't quite tell why jakartaee needs to be handled differently here.
Maybe a comment with examples would make it clear?

Reply:
We use these same ant tasks to create jakartaeetck.zip (& jakartaee-smoke.zip) bundles. I did not want to apply the TCK bundle name changes to those bundles now.  This change is to continue using the old names for those bundles.